controller/drivers/rpi/pwm.go

```diff
@@ -0,0 +1,42 @@
+package rpi
+
+import (
+	"fmt"
+
+	"github.com/reef-pi/reef-pi/controller/types/driver"
+
+	"github.com/reef-pi/rpi/pwm"
+)
+
+type rpiPwmChannel struct {
+	channel int
+	name    string
+	pwm     *rpiPwm
+}
```

Comment:
pwm *rpiPwm should not be a member, its convoluted . lets call this just PWMChannel, and PWMDriver

Reply:
Fixed